src/transformers/models/altclip/modeling_altclip.py

```diff
@@ -844,9 +844,9 @@ def __init__(self, config: AltCLIPConfig):
         super().__init__()
         self.embed_dim = config.hidden_size
         self.self_attn = AltCLIPAttention(config)
-        self.layer_norm1 = nn.LayerNorm(self.embed_dim)
+        self.layer_norm1 = nn.LayerNorm(self.embed_dim, eps=config.layer_norm_eps)
```

Comment:
A few lines above, we have `def __init__(self, config: AltCLIPConfig):`. This is incorrect. It should be `AltCLIPVisionConfig`. Same comment applies for `CLIP`, where it should be `the union of CLIPTextConfig and CLIPVisionConfig`.

I decide not to address this issue in this PR though - a bit too much involved.